common/static/coffee/spec/discussion/view/discussion_thread_list_view_spec.coffee

```diff
@@ -6,17 +6,17 @@ describe "DiscussionThreadListView", ->
         appendSetFixtures("""
         <script type="text/template" id="thread-list-template">
             <div class="forum-nav-header">
-                <a href="#" class="forum-nav-browse" aria-haspopup="true">
-                    <i class="icon fa fa-bars"></i>
+                <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
+                    <span class="icon fa fa-bars" aria-hidden="true"></span>
                     <span class="sr">Discussion topics; current selection is: </span>
```

Comment:
How about if we change the SR text from "current selection is:" to "currently listing:"  I feel like that is a more accurate representation.